pkg/controller/clusterprovision/clusterprovision_controller_test.go

```diff
@@ -128,12 +128,14 @@ func TestClusterProvisionReconcile(t *testing.T) {
 			expectedFailReason: unknownReason,
 		},
 		{
-			name: "keep job after success",
+			name: "removed job after success",
```

Comment:
Re-work this test case so that we test keeping the job within 24 hours and deleting the job after 24 hours.